### PR TITLE
remove state.pods; fix runtime & results

### PR DESCRIPTION
**Pull request by lihebi**
This is follow-up PR for #331 to fix broken things.

What's changed:

- state.pods are removed and fixed all compile errors
- Runtime and results are fixed.

Remaining TODOs regarding this centered-Yjs transition:

- [ ] fix copy-n-paste
- [ ] fix jupyter import/export
- [ ] save results to Yjs and sync with peers
- [ ] sync titles: doc title, scope title, pod title

### CODE REVIEW

File: ui/src/components/Sidebar.tsx [hunk 1]
```diff
@@ -876,7 +878,10 @@ function ExportJupyterNB() {
   const store = useContext(RepoContext);
   if (!store) throw new Error("Missing BearContext.Provider in the tree");
   const repoName = useStore(store, (state) => state.repoName);
-  const pods = useStore(store, (state) => state.pods);
+  const nodesMap = useStore(store, (state) => state.getNodesMap());
+  const nodes = Array.from<ReactflowNode>(nodesMap.values());
+  const podResults = useStore(store, (state) => state.podResults);
+  const codeMap = useStore(store, (state) => state.getCodeMap());
```
Comment (senwang86) [inline ui/src/components/Sidebar.tsx:884]: Is the `podResults` not synced with YJS by design?

Comment (lihebi) [reply ui/src/components/Sidebar.tsx:884]: It should be saved to Yjs. It's on my TODO list.

Comment (senwang86) [reply ui/src/components/Sidebar.tsx:884]: > It should be saved to Yjs. It's on my TODO list.

Is there any concern to append `podResults` to `codeMap`?

Comment (lihebi) [reply ui/src/components/Sidebar.tsx:884]: That's an option; I'm thinking about it.

**Pull request merged**